frontend/javascripts/main.js

```diff
@@ -48,7 +52,9 @@ document.addEventListener("DOMContentLoaded", async () => {
   if (containerElement) {
     ReactDOM.render(
       <Provider store={Store}>
-        <Router />
+        <DndProvider backend={dndBackend}>
```

Comment:
Was this fix documented somewhere or how did you come up with it?

Reply:
Yes, the bug is somewhat known. However, the solution is usually recommended for scenarios where multiple DnD components are used in parallel. Here's a brief mention of the technique: https://github.com/frontend-collective/react-sortable-tree#if-it-doesnt-work-with-other-components-that-use-react-dnd

I also added a code comment which points to the specific template I used for the fix.